src/newLaunch/lbp/stages/stage4.ts

```diff
@@ -291,6 +291,8 @@ export class Stage4 extends BaseStage<ILbpConfig> {
     } else if (!(Number.parseInt(endTimes[1]) >= 0)
       || !(Number.parseInt(endTimes[1]) < 60)) {
       message = "Please enter a valid value for End Time";
+    } else if (!Utils.isAddress(this.launchConfig.launchDetails.adminAddress)) {
+      message = "Please enter a valid value for launch administrator";
```

Comment:
Lets change the text to : "Please enter a valid address for the LBP administrator"

Reply:
I changed to : "Please enter a valid wallet address for LBP Administrator", like the seed wizard

Reply:
ok, but for correct english, it should be "the LBP Administrator", in both places ("the Seed Administrator" for seeds).

Reply:
Done